app/src/config.ts

```diff
@@ -0,0 +1,10 @@
+export default {
+  ipfs: {
+    endpoint: 'https://ipfs-api.dev.fleek.cool',
+    headers: {
+      Authorization: `v2 ${import.meta.env.VITE_FLEEK_API_KEY}`, // or Bearer <JWT> or public <AppKey>
+    },
+    metaPtrEndpoint: 'https://ipfs-api.dev.fleek.cool',
+    // metaPtrEndpoint: 'https://cloudflare-ipfs.com/ipfs',
+  },
+};
```

Comment:
^ I'm assuming this would be a diff config for prod / dev. If yes would it make sense doing something like `dev: { ipfs : {...}}`

Reply:
or we could just define these in env files

Reply:
Similarly, I think we can just delete this file and move the contents to `app/src/utils/ipfs.ts`, since that's the only place this is used. That way all IPFS config/abstraction lives in one place which is nice

Reply:
followed @mds1 suggestion